cmd/cmd/root.go

```diff
@@ -29,7 +29,7 @@ import (
 var cfgFile string
 var verbose bool
 
-var programName = os.Args[0]
+var programName = "mi"
```

Comment:
We shouldn't change this. This is done to make the cli name dynamic. 
@SanojPunchihewa